crates/net/downloaders/src/bodies/bodies.rs

```diff
@@ -515,6 +523,7 @@ impl Default for BodiesDownloaderBuilder {
             stream_batch_size: 1_000,
             max_buffered_blocks_size_bytes: 2 * 1024 * 1024 * 1024, // ~2GB
             concurrent_requests_range: 5..=100,
+            max_queued_bodies_size: 100,
```

Comment:
I think we want to limit this by using the targeted stream_batch_size, so we cant misconfigure this

I think a proper cutoff could be `self.queued_bodies.len() < 2 * stream_batch_size`

Reply:
good idea.

Reply:
I have squashed the commits and update with the idea.